src/unasync/__init__.py

```diff
@@ -4,7 +4,7 @@
 import tokenize as std_tokenize
 from tokenize import NAME, NEWLINE, NL, STRING, ENCODING
 
-from setuptools.command.build_py import build_py
+from setuptools.command import build_py as orig
```

Comment:
Maybe use a more informative name than `orig` and also add a comment here why we are doing it.

Reply:
In fact I simply copied the code from setuptools because they themselves subclass from distutils as explained in the commit: https://github.com/pypa/setuptools/blob/master/setuptools/command/build_py.py#L3. Which is why I used "orig". I could also have used "_build_py".

I added a commit to clarify, do you think it's better like this?